### DISCUSSION TITLE
Ctrl + Enter should go to new line instead opening copilot

### DISCUSSION THREAD

**Discussion by SarkarKurdish** [community/community#7412] [Codespaces] [answered]
Hey, thanks for this nice extension, one slight problem is, we are in a middle of a line and press (ctrl + enter) the cursor would be in a new line like this

1: this is just(the cursor is here) a line

 so you might get why this key binding is important and we really got use to it, after copilot this key binding now opens copilot and there seems to be no way changing this binding,

Thanks for help.

**Comment by mh-antor** [comment, marked as answer]
Here is the solution worked for me.
https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md

**Reply by SarkarKurdish** [reply]
thanks, soon after i opened this issue i was able to change the key combination in the Key Binding Setting

**Reply by dovigz** [reply]
Thanks worked for me I added in my settings.json and restarted vs code
`"github.copilot.inlineSuggest.enable": true,`

**Reply by cprosche** [reply]
This doc 404s 😢

**Comment by brandonporcel** [comment]
I solved it like this

File -> Preferences -> Keyboard Shortcuts -> Search "ctrl enter" -> Edit github copilot shortcut

**Reply by Marc-MMA** [reply]
That's what we want! Answer A1! Thanks

**Reply by codekiln** [reply]
Any suggestions? What did you mark it as?

**Reply by rijaulsk** [reply]
I just deleted the shortcut, thanks for the solution.

**Reply by dipesh20-Y** [reply]
Thanks; it worked for me as well

**Reply by mohammad-zrar** [reply]
Perfect
